pkgs/development/libraries/gdal/default.nix

```diff
@@ -20,14 +20,13 @@ composableDerivation.composableDerivation {} (fixed: rec {
 
   hardeningDisable = [ "format" ];
 
-  # Don't use optimization for gcc >= 4.3. That's said to be causing segfaults.
   # Unset CC and CXX as they confuse libtool.
-  preConfigure = "export CFLAGS=-O0 CXXFLAGS=-O0; unset CC CXX";
```

Comment:
I did remove the no-optimization flags, because I didn't think they were necessary. "-O0" probably decreases performance quite a bit. They are not necessary for homebrew on OS X: https://github.com/Homebrew/homebrew-core/blob/master/Formula/gdal.rb and the initial reason to add those flags "Don't use optimization for gcc >= 4.3. That's said to be causing segfaults." I quite doubt. I did a quick test of the binaries and for what it's worth they seemed to work fine for me.